sql/size_utils.sql

```diff
@@ -74,7 +74,7 @@ RETURNS TABLE (
     index_bytes bigint,
     toast_bytes bigint,
     total_bytes bigint)
-LANGUAGE PLPGSQL STABLE STRICT AS
+LANGUAGE PLPGSQL VOLATILE STRICT AS
```

Comment:
they are all PARALLEL SAFE are they not?

Reply:
I do not think these are parallel safe as we potentially access other data nodes to get the information.